bash-test

```diff
@@ -93,7 +93,7 @@ time {
     done
 
     PATH=$original_path
-    unset -f $test_names
+    unset -f "$test_names"
```

Comment:
Hi Urs, that is the only case where we actually __want__ to have word splitting.
I'd suggest that instead of wrapping it on quotes, we just tell shellcheck to don't complain about quotes in this particular line by adding a comment above it:

```
# shellcheck disable=SC2086
unset -f $test_names
```